src/main/java/frc/robot/Robot.java

```diff
@@ -42,8 +45,9 @@ public void robotInit() {
     // yet. Thus, their requires() statements may grab null pointers. Bad
     // news. Don't move it.
     this.m_oi = new OI();
+    this.m_pigeon = new PigeonHandler(m_drive.getTalon(RobotMap.m_motorFrontLeft));
```

Comment:
LIKELY BUG: You configure to get talon from FRONTLEFT but then in Robot.java:teleop_init(), you are configuring it the FrontRight